app/controllers/api/v1/invoices_controller.rb

```diff
@@ -27,7 +27,7 @@ def show
 
       def index
         invoices = current_organization.invoices
-        invoices = invoices.where(customer_id: params[:customer_id]) if params[:customer_id]
+        invoices = invoices.joins(:customer).where(customers: { external_id: params[:external_customer_id] }) if params[:external_customer_id]
```

Comment:
Layout/LineLength: Line is too long. [142/120] (https://rubystyle.guide#max-line-length)